web_src/js/features/common-form.ts

```diff
@@ -13,12 +15,13 @@ export function initGlobalFormDirtyLeaveConfirm() {
 export function initGlobalEnterQuickSubmit() {
   document.addEventListener('keydown', (e) => {
```

Comment:
```suggestion
  document.addEventListener('keydown', (e: KeyboardEvent & {target: HTMLElement}) => {
```

This avoids having to declare a variable and you can use `e.target` as-is.

Reply:
I have no motivation to do that, that's just personal preference.

If you strongly prefer to, feel free to edit this PR directly.